go/oppobloom/sizetrackingfilter.go

```diff
@@ -0,0 +1,59 @@
+package oppobloom
+
+import "sync/atomic"
+
+// Like Filter, but has additional ability to expose the number of items currently
+// in the filter, as well as the total size of the data held by the filter
+type SizeTrackingFilter struct {
```

Comment:
Ah, I was trying to leave this lib as small as possible and have you keep this code in your codebase. I'm not sure that's as general was we'd assume.